graphql/schema/types/repo_ref.graphql

```diff
@@ -18,6 +18,7 @@ input RepoRefInput {
   gitTagVersionsEnabled: Boolean
   manualPrTestingEnabled: Boolean
   notifyOnBuildFailure: Boolean
+  oldestAllowedMergeBase: String
```

Comment:
do we intend to show this field for repos? (I guess the question I'm asking is, is it valid to set this field on a repo?)